app/assets/javascripts/admin/addon/controllers/admin-dashboard.js

```diff
@@ -106,16 +96,7 @@ export default class AdminDashboardController extends Controller {
     });
 
     AdminDashboard.fetchProblems()
-      .then((model) => {
-        this.set(
-          "highPriorityProblems",
-          model.problems.filterBy("priority", "high")
-        );
-        this.set(
-          "lowPriorityProblems",
-          model.problems.filterBy("priority", "low")
-        );
-      })
```

Comment:
Partitioning this into low and high is now a responsibility of `DashboardProblems`.